# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/nsITouchBarHelper.idl
# Commit: 64d7f0db63a4
# Full Hash: 64d7f0db63a45ec431847e77c6220f5d2ef9c4a9
# Author: harry <htwyford@mozilla.com>
# Date: 2019-10-11 09:30:10
# Regressor Bug: 1563350
# File Overlap Count: 2
# Description:
#   Bug 1563350 - Add popover to the Touch Bar that displays when the Urlbar has focus. r=mikedeboer,spohl,fluent-reviewers,Pike,flod
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D38563
# ==============================================================================

diff -r 879c857b5b65 -r 64d7f0db63a4 widget/nsITouchBarHelper.idl
--- a/widget/nsITouchBarHelper.idl	Wed Oct 09 10:46:05 2019 +0000
+++ b/widget/nsITouchBarHelper.idl	Thu Oct 10 19:57:32 2019 +0000
@@ -23,6 +23,11 @@
   readonly attribute AString activeTitle;
 
   /**
+   * Return true if the Urlbar has focus.
+   */
+  readonly attribute boolean isUrlbarFocused;
+
+  /**
    * Returns all available Touch Bar Inputs in an nsIArray
    * of nsITouchBarInput objects.
    */
